xcube/core/store/typespecifier.py

```diff
@@ -74,6 +75,40 @@ def __eq__(self, other) -> bool:
             return False
         return len(self.flags.symmetric_difference(other_type.flags)) == 0
 
+    def satisfies(self, other: Union[str, "TypeSpecifier"]) -> bool:
+        """
+        Tests whether this type specifier satisfies (the requirements of) another type specifier.
+
+        A type specifier A satisfies a type specifier B if
+        * if the type names of A and B are equal
+        * and
+          - if B has no flags
+          - or if all the flags of B are a subset of the ones of A.
+
+        :param other: Another type specifier, as string or *TypeSpecifier*.
+        :return: Whether this type specifier satisfies another type specifier.
+        """
+        other_type = self.normalize(other)
+        if self.name == '*' or other_type.name == '*':
+            return True
+        if self.name != other_type.name:
+            return False
+        if not other_type.flags:
+            return True
+        return len(other_type.flags.difference(self.flags)) == 0
```

Comment:
```suggestion
        return other_type.flags.issubset(self.flags)
```

Logically equivalent, but I find this formulation far more self-explanatory.